### ISSUE TITLE
Add pyproject.toml and poetry

### ISSUE THREAD

**Issue by kigawas** [darosior/python-bip32#4]
This library is handy and I wrote a [bip44 version](https://github.com/kigawas/python-bip44) 😃 

Perhasp you can introduce pyproject.toml with poetry

**Comment by darosior** [comment]
Yes, a PR to migrate to the new standard way of managing Python dependencies is welcome. :-)

**Comment by darosior** [comment]
We finally migrated to the `pyproject.toml` standard so i'm going to close this issue. If you still think this project should use Poetry, please open a dedicated issue motivating it.

**Comment by kigawas** [comment]
it's been five years...

**Comment by darosior** [comment]
It has!
